src/main/java/com/musala/sdcs/device/channel/type/PlayControlType.java

```diff
@@ -8,4 +8,21 @@
     Play,
     Pause,
     Stop;
+	
+	private static final String CMD_PLAY_STRING = "Play";
+	private static final String CMD_PAUSE_STRING = "Pause";
+	private static final String CMD_STOP_STRING = "Stop";
+	
+	public static PlayControlType getCommandValueFromString(String type) {
+		switch (type) {
+			case CMD_PLAY_STRING:
+				return Play;
+			case CMD_PAUSE_STRING:
+				return Pause;
+			case CMD_STOP_STRING:
+				return Stop;
+		}
+		
+		return null;
```

Comment:
In order not to return `null` here, please define a `default` case. Inside it, a new state with value `undef` will be returned. On the other hand, the `undef` state will be defined as a constant in `ChannelCreator`.  The `undef` state will be a legal state for a device and, if it is set in the DB, the UI will react in a certain manner (we will discuss).